### ISSUE TITLE
How many pages or size of content can this lib handle?

### ISSUE THREAD

**Issue by tisonkun** [easyops-cn/docusaurus-search-local#435]
AFAIK local search runs within the browser and thus is limited with the resource available in browser.

Is there any number about how many pages or sizes of content this lib can handle?

For example, Pagefind said:

> The goal of Pagefind is that websites with tens of thousands of pages should be searchable by someone in their browser, while consuming as little bandwidth as possible. Pagefind’s search index is split into chunks, so that searching in the browser only ever needs to load a small subset of the search index. Pagefind can run a full-text search on a 10,000 page site with a total network payload under 300kB, including the Pagefind library itself. For most sites, this will be closer to 100kB.

**Comment by weareoutman** [comment]
You're correct, local search has such limitations. Currently we have no such recommended number, would like to figure it out later.

Making the search index splitted like Pagefind you mentioned above, seems to be a nice approach for us too.
